doc/reference/releases-snap.md

```diff
@@ -0,0 +1,127 @@
+
+(ref-releases-lts-support)=
+#### Support
+
+LTS releases receive standard support for five years, meaning that it receives continuous updates according to the support levels described below. An [Ubuntu Pro](https://ubuntu.com/pro) subscription can provide additional support and extends the support duration by an additional five years.
+
+(ref-releases-lts-support-levels)=
+#### Support levels
+
+Standard support for LTS releases starts at the full support level, then moves to maintenance support, then extended support. Once it reaches End of Life (EOL), that means it no longer receives any updates.
+
+- **Full support**: Some new features, frequent bugfixes, and security updates are provided.
+- **Maintenance support**: High impact bugfixes and security updates are provided.
+- **Extended support**: Only critical security updates are provided.
+
+(ref-releases-lts-support-current)=
+#### Currently supported
+
+The currently supported LTS releases are 5.21._z_ and 5.0._z_.
+
+- 5.21._z_ is supported until June 2029.
+  - Currently in full support phase.
+  - Updates are provided approximately every six months. This schedule is an estimate that can change based on priorities and discovered bugs.
+- 5.0._z_ is supported until June 2027.
+  - Currently in extended support phase.
+  - Updates are provided when critical security updates are needed.
+
+(ref-releases-feature)=
+### Feature releases
+
+Feature releases are pushed out more often and contain the newest features and bugfixes. Since they are less tested than LTS releases, they are **not recommended for production use**.
+
+These releases follow the format _x.y_, and they never include a point number _z_. Currently, feature releases for LXD are numbered {{current_feature_track}}._y_, with _y_ incrementing for each new release. Every two years, the latest feature release becomes an LTS release.
+
+#### Support
+
+Feature releases receive continuous updates via each new release. The newest release at any given time is also eligible for additional support through an [Ubuntu Pro](https://ubuntu.com/pro) subscription.
+
+(ref-snap)=
+## The LXD snap
+
+The recommended way to install LXD is [its snap package](https://snapcraft.io/lxd), if snaps are available for your system. A key benefit of snap packaging is that it includes all required dependencies. This allows LXD to run in a consistent environment on many different Linux distributions. Using the snap also streamlines updates through its [channels](https://snapcraft.io/docs/channels).
+
+(ref-snap-channels)=
+### About channels
+
+Each installed LXD snap follows a [channel](https://snapcraft.io/docs/channels). Channels have a track and a risk level (for example, `6/stable`). Each channel points to one release at a time, and when a new release is published to a channel, it replaces the previous one. {ref}`Updating the snap <howto-snap-updates>` then updates to that release.
```

Comment:
In `6/stable` here can we utilise the `current_feature_track` variable below so it auto updates?